.github/workflows/smoke-test.yml

```diff
@@ -21,10 +21,6 @@ jobs:
 
       - name: Checkout
```

Comment:
Could we lose this bit too?

Reply:
I was assuming it'd still need to checkout the repo, in order to get the actual tests?

Reply:
ah i see — sorry, thought this was the action for the main bit, ignore me!